src/types/rigid-body.ts

```diff
@@ -10,6 +10,7 @@ export type RigidBodyType =
   | 'kinematicVelocity'
   | 'fixed'
 
+export interface enableBolean { x: boolean, y: boolean, z: boolean }
```

Comment:
I think we can move this one to a separate type file to keep the `rigid-bosy.ts` file clean

something like `boolean.ts` or `object.ts`